core/src/main/kotlin/io/specmatic/conversions/EnvironmentAndPropertiesConfiguration.kt

```diff
@@ -1,62 +1,70 @@
 package io.specmatic.conversions
 
+import io.specmatic.core.SpecmaticConfig
 import org.apache.commons.lang3.BooleanUtils
 
-data class EnvironmentAndPropertiesConfiguration(val environmentVariables: Map<String, String>, val systemProperties: Map<Any?, Any?>) {
-    constructor() : this(System.getenv(), System.getProperties().toMap())
+data class EnvironmentAndPropertiesConfiguration(
```

Comment:
The name of this class no more describes what it's doing. It reads all Specmatic configuration, which includes the config file, properties and environment variables.

How about renaming it to `Configuration`?

Note: there already is another class named `Configuration`, which may be more appropriately renamed `ConfigurationConstants`. In fact, there are methods and properties in this class related to the Specmatic config file, which ideally could move to SpecmaticConfig. The `config` and `_config` properties could be moved easily into `SpecmaticConfig`, as they are referred to in just a few places. The variables named `innerGlobalConfigFileName`,  methods `getGlobalConfigFileName`, `getConfigFileName`, and property `globalConfigFileName` could be moved later on as that moving them may be a bit more complicated.